packages/desktop-client/src/components/accounts/Account.tsx

```diff
@@ -1848,6 +1874,7 @@ function AccountHack(props: AccountHackProps) {
 
   return (
     <AccountInternal
+      dispatch={dispatch}
```

Comment:
Any chance we can just call `useDispatch` inside `AccountInternal` instead? This component already has so many props 😭

Reply:
Unfortunately not :( The AccountInternal is a class component and we can't use hooks inside. I've been trying to migrate it to a functional component but there hasn't been much movement on the PR because it's hard to review due to the component already being massive as it is. It's hard to break down the chunks into smaller chunks.